File: Desktop/application/backend/entity/Parcela.java
package com.example.demo.entity;


import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder

public class Parcela {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column
    (nullable = true)
    private byte[] ln;


    @Column(name = "br_kat_parcele", nullable = false)
    private String brKatParcele;

    @Column(name = "kat_opstina", nullable = false)
    private String katOpstina;

    @Column(
            nullable = false
    )
    private String opstina;

}

File: Desktop/application/frontend/models/Parcela.ts
export class Parcela{
    id: number
    brKatParcele:string
    katOpstina:string
    opstina:string
    ln: Array<number> // ili Array<number>

    constructor() {
        this.id = 0;
        this.brKatParcele = '';
        this.katOpstina = '';
        this.opstina = '';
        this.ln = null;
    }
    
}